knora/dsplib/utils/excel_to_json_properties.py

```diff
@@ -30,75 +33,110 @@ def validate_properties_with_schema(json_file: str) -> bool:
     return True
 
 
-def properties_excel2json(excelfile: str, outfile: str) -> list[dict[str, Any]]:
+def _row2prop(row: pd.Series, row_count: int, excelfile: str) -> dict[str, Any]:
     """
-    Converts properties described in an Excel file into a properties section which can be integrated into a DSP ontology
+    Method that takes a row from a pandas DataFrame, reads its content, and returns a dict object of the property
 
     Args:
-        excelfile: path to the Excel file containing the properties
-        outfile: path to the output JSON file containing the properties section for the ontology
+        row: row from a pandas DataFrame that defines a property
+        row_count: row number of Excel file
+        excelfile: name of the original excel file
 
     Returns:
-        List(JSON): a list with a dict (JSON) for each row in the Excel file
+        dict object of the property
     """
-    # load file
-    wb = load_workbook(filename=excelfile, read_only=True)
-    sheet = wb.worksheets[0]
-    props = [row_to_prop(row) for row in sheet.iter_rows(min_row=2, values_only=True, max_col=13)]
-
-    prefix = '"properties":'
 
-    if validate_properties_with_schema(json.loads(json.dumps(props, indent=4))):
-        # write final list to JSON file if list passed validation
-        with open(file=outfile, mode='w+', encoding='utf-8') as file:
-            file.write(prefix)
-            json.dump(props, file, indent=4)
-            print('Properties file was created successfully and written to file:', outfile)
-    else:
-        print('Properties data is not valid according to schema.')
-
-    return props
+    name = row["name"]
+    supers = [s.strip() for s in row["super"].split(",")]
+    _object = row["object"]
 
+    labels = {}
+    if row.get("en"):
+        labels["en"] = row["en"]
+    if row.get("de"):
+        labels["de"] = row["de"]
+    if row.get("fr"):
+        labels["fr"] = row["fr"]
+    if row.get("it"):
+        labels["it"] = row["it"]
+    if row.get("rm"):
+        labels["rm"] = row["rm"]
```

Comment:
These could be nicely simplified. (same for the following, I just type out this one)

```suggestion
    languages = {"en", "de", "fr", "it", "rm"}
    labels = {lang: row[lang] for lang in languages if lang in row}
```